### DISCUSSION TITLE
What is Image or Images?

### DISCUSSION THREAD

**Discussion by grandwaterman** [rnmapbox/maps#3072] [Q&A] [answered]
Is the image here a snapshot of the map? I worked with mapbox react, there is an api to display an image of the map snapshot, is this the same thing here? thanks

**Comment by mfazekas** [comment, marked as answer]
Image and Images are for [resolvedImages](https://docs.mapbox.com/style-spec/reference/types/#resolvedimage). Like [icon-image](https://docs.mapbox.com/style-spec/reference/layers/#layout-symbol-icon-image) in symbol layer - it's equivalent to `addImage` in mapbox-gl-js.

https://docs.mapbox.com/mapbox-gl-js/example/add-image/

**Reply by grandwaterman** [reply]
Thanks @mfazekas !
